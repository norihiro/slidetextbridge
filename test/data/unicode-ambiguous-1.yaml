steps:
  - type: stdin

  - type: linebreak
    # ambiguous_char_width: 1 # default
    split_long_line: 16

  - type: stdout
    page_delimiter: "\n"
