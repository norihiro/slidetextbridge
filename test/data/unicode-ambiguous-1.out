6Ωの抵抗と3Ωの抵
抗を並列に接続す
ると、2Ωになりま
す。
